### PR TITLE
demo: CSI example for DigitalOcean block storage

### PR DESCRIPTION
**Pull request by tgross**
This is a slight rework of an internal testing deployment.

### CODE REVIEW

File: demo/csi/digitalocean/main.tf [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  external_id           = digitalocean_volume.test_volume.id
+  access_mode           = "single-node-writer"
+  attachment_mode       = "block-device"
+  deregister_on_destroy = true
+}
+
+# consume the volume
+resource "nomad_job" "redis" {
+  jobspec    = templatefile("${path.module}/volume-job.nomad", { volume_id = nomad_volume.test_volume.id })
+  depends_on = [nomad_volume.test_volume]
```
Comment (cgbaker) [inline demo/csi/digitalocean/main.tf:34]: is this necessary, considering that the jobspec above relies on the computed field `id` from the `nomad_volume.test_volume` resource?

Comment (cgbaker) [reply demo/csi/digitalocean/main.tf:34]: similar to https://github.com/hashicorp/team-nomad/blob/master/terraform/csi_volumes/aws/env/us-east/main.tf#L76-L80

**Review by cgbaker — APPROVED** [review]
🏆

File: demo/csi/digitalocean/main.tf (hunk 1 above)
Comment (tgross) [reply demo/csi/digitalocean/main.tf:33]: good catch!

**Pull request merged**